src/filters/interframe.cpp

```diff
@@ -20,7 +20,8 @@ static uint8_t *frm3 = NULL;
 
 extern uint32_t qRGB_COLOR_MASK[2];
 
-static void Init()
```

Comment:
Just so this does not go unnoticed, this `Init()` function was declared static at `interframe.hpp`. That means that every place importing `interframe.hpp` needs to code the `Init()` function. This was not being done at any other file which imports `interframe.hpp`, hence the decision here.

Reply:
I think it would be best to rename this function to something like `interframe_filter_init()` so as not to pollute the namespace with an `Init()` function of questionable origin.

I just remembered this today too.